### PR TITLE
Refactor project structure

### PR DESCRIPTION
**Pull request by rabin245**
Refactored the structure of the components.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
-  </head>
-  <body>
-    <div id="root"></div>
-    <script type="module" src="/src/main.jsx"></script>
-  </body>
-</html>
+
+<head>
+  <meta charset="UTF-8" />
+  <link rel="icon" type="image/svg+xml" href="/vite.svg" />
```
Comment (evilcba) [inline index.html:6]: add your own favicon

**Comment by rojandahal** [comment]
LGTM

**Review by rojandahal — APPROVED** [review]

**Review by evilcba — CHANGES_REQUESTED** [review]
structure your folders and files

File: index.html (hunk 1 above)
Comment (rabin245) [reply index.html:6]: I have not found a good enough icon for that.

**Review by evilcba — APPROVED** [review]
lgtm

**Review by aayussraut — COMMENTED** [review]
LGTM

**Pull request merged**